eslint.config.js

```diff
@@ -0,0 +1,13 @@
+import { addExtensions, litConfig, setDirectoryConfigs, testingConfig } from 'eslint-config-brightspace';
+
+export default [
+	{ ignores: ['.vdiff/'] },
+	...addExtensions(setDirectoryConfigs(
+		litConfig,
+		{ test: testingConfig },
+	), ['.js', '.html']),
+	{
+		files: ['demo/**/**.html'],
```

Comment:
This is already in the base `litConfig`, ya?

Reply:
Not really, in the shared config any file in a `demo` directory is targeted but not recursively in any child directory.